apps/remix-ide/src/app/panels/terminal.js

```diff
@@ -746,10 +746,15 @@ class Terminal extends Plugin {
       }
     }
     try {
-      await this.call('scriptRunner', 'execute', script)
+      if (script.trim().indexOf('git') === 0) {
+        const result = await this.call('git', 'command', script)
+        self.commands.html(yo`<pre>${result}</pre>`)
+      } else {
+        await this.call('scriptRunner', 'execute', script)
```

Comment:
Why not displaying the result here ? I guess it's because scriptRunner emit an event instead of returning a value.
It's quite difficult to follow and I think you should take the opportunity of this PR to homogenize the process. Ideally we should have : 
```javascript
const pluginName = {
  remix: 'scriptRunner',
  git: 'git'
}
const prefix = script.trim.split('.').unshift()
const plugin = pluginName[prefix]
const result = await this.call(plugin, 'command', script)
```

Reply:
that gonna be hard to make `scriptRunner` to return a value, as we don't really know when a script finishes (it can `console.log` at any time). will try to update git to emit event instead.

Reply:
You can increase the timeout of the scriptRunner plugin and return the value. It doesn't prevent you to listen on log events if you need